test/references/JSONLD_Web Kaggle.txt

```diff
@@ -0,0 +1,20 @@
+register(DataDep(
+    "Map of Co-Seismic Landslides  for the M 7.8 Kaikoura, New Zealand Earthquake",
+    """
+    	Dataset: Map of Co-Seismic Landslides  for the M 7.8 Kaikoura, New Zealand Earthquake
+	Website: https://zenodo.org/record/1287281
+	Author: Valkaniotis Sotiris et al.
+	License: https://creativecommons.org/licenses/by/4.0/
+	Date: December 20, 2016
+	
+	<p>Prepared by the Research Group on Earthquake Geology in Greece (http://eqgeogr.weebly.com/)</p>
```

Comment:
Looks like we need to strip any HTML out of the description.
We have Gumbo to parse HTML  and the `text_only` method already.
So should be easy to add a function in `utils.jl` for that